htsinfer/utils.py

```diff
@@ -0,0 +1,36 @@
+"""Utilities used across multiple HTSinfer modules."""
+
+from pandas import DataFrame  # type: ignore
+
+
+def results_validator(
+    data: DataFrame,
```

Comment:
I think this function would make more sense if you pass it the column as a vector rather than passing the dataframe. It'd be much more elegant and generic.